addon/components/polaris-color-picker.js

```diff
@@ -63,97 +64,117 @@ export default Component.extend({
 
     const backgroundColor = `rgba(${red}, ${green}, ${blue}, ${alpha})`;
     return htmlSafe(`background-color: ${backgroundColor};`);
-  }).readOnly(),
+  }
 
   /**
    * @private
    */
-  draggerX: computed('color.saturation', 'pickerSize', function() {
+  @(computed('color.saturation', 'pickerSize').readOnly())
+  get draggerX() {
     const {
       color: { saturation },
       pickerSize,
     } = this.getProperties('color', 'pickerSize');
     return clamp(saturation * pickerSize, 0, pickerSize);
-  }).readOnly(),
+  }
 
   /**
    * @private
    */
-  draggerY: computed('color.brightness', 'pickerSize', function() {
+  @(computed('color.brightness', 'pickerSize').readOnly())
+  get draggerY() {
     const {
       color: { brightness },
       pickerSize,
     } = this.getProperties('color', 'pickerSize');
     return clamp(pickerSize - brightness * pickerSize, 0, pickerSize);
-  }).readOnly(),
+  }
+
+  @computed('contentId')
+  get element() {
+    return document.querySelector(`#${this.contentId}`);
+  }
 
   didRender() {
-    this._super(...arguments);
+    super.didRender(...arguments);
```

Comment:
I don't think we need `didRender` anymore...actually we want to stop using any other lifecycle hooks except init/destroy....instead we should have this logic inside the action you added `setContentId` (renamed ofc)